용재/1406.py

```diff
@@ -0,0 +1,27 @@
+# https://www.acmicpc.net/problem/1406
+# 에디터
+
+getText = list(input())
+
+N = int(input())
+
+inputNumbers = [str(input()) for _ in range(N)]
+
+L = len(getText)
+
+restText = []
+
+for i in inputNumbers:
+  if i[0] == "L":
+    if len(getText) > 0:
```

Comment:
```suggestion
  if i[0] == "L" and getText:
```
if 문을 두번 중첩해서 쓰는 것보다 `and`연산자를 사용해서 depth를 줄이는 게 좋을 것 같아요! 
`getText`라는 리스트가 비어있는지 확인하는 것도 `len()`함수로 길이를 보는 방법보다 위 방법이 좋은 것 같습니다!